README.md

```diff
@@ -258,23 +307,45 @@ You can then add the generated information and the `authorization` parameter to
 ]
 ```
 
-### Amazon SageMaker
+### Amazon
+
+#### SageMaker
 
 You can also specify your Amazon SageMaker instance as an endpoint for chat-ui. The config goes like this:
 
 ```env
 "endpoints": [
     {
-      "host" : "sagemaker",
-      "url": "", // your aws sagemaker url here
+      "type" : "aws",
+      "service" : "sagemaker"
+      "url": "",
       "accessKey": "",
       "secretKey" : "",
-      "sessionToken": "", // optional
-      "weight": 1
+      "sessionToken": "",
+      "weight": 1,
```

Comment:
json probably disallows trailing commas

Reply:
Yep you're right, thanks for catching it